cmd2/parsing.py

```diff
@@ -607,6 +607,16 @@ def _command_and_args(tokens: List[str]) -> Tuple[str, str]:
 
         return command, args
 
+    @staticmethod
+    def shlex_split(str_to_split: str) -> List[str]:
```

Comment:
This doesn’t quite feel right to me as a static method in StatementParser.  What would you think about making it a standalone function in either parsing.py (same file)?